Homework 4/lesson_4.py

```diff
@@ -0,0 +1,69 @@
+from database_handler import execute_query
+from flask import Flask
+from webargs.flaskparser import use_kwargs
+from validator import order_price_validator, track_validator
+
+app = Flask(__name__)
+
+
+@app.route("/order-price")
+@use_kwargs(order_price_validator, location='query')
+def order_price(country) -> str:
+    """The function displays total sales value from chinook.db.
+    If the Country value is specified, the function returns the total value for that country."""
+
+    query = (f"SELECT BillingCountry, SUM(UnitPrice * Quantity) AS SALES FROM invoice_items LEFT JOIN invoices "
+             f"ON invoice_items.InvoiceId = invoices.InvoiceId")
+
+    if country:
+        country = country.title()
+        query += f" WHERE BillingCountry = '{country}'"
+        records: list = execute_query(query)
+        country_name: str = records[0][0]
```

Comment:
this code is some way duplicated with 
 records: list = execute_query(query)
  total_sales: float = records[0][1]
return f"Total sales: {total_sales}"

Unify it and and in "if" just add where SQL part